extensions/vertx-http/runtime/src/test/java/io/quarkus/vertx/http/runtime/logging/RequestLogMessageFormatterTest.java

```diff
@@ -0,0 +1,37 @@
+package io.quarkus.vertx.http.runtime.logging;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+
+import io.vertx.core.http.HttpVersion;
+
+class RequestLogMessageFormatterTest {
+
+    @Test
+    void format() {
+        final RequestLogMessageFormatter formatter = new RequestLogMessageFormatter();
+
+        final Map<String, String> headers = new HashMap<>();
+        headers.put("Cookie", "any-cookie");
+        headers.put("Accept", "application/json, text/plain, */*");
+        headers.put("Connection", "keep-alive");
+
+        final RequestLogMessage requestLogMessage = RequestLogMessage.builder()
+                .headers(headers)
+                .method("GET")
+                .path("/quarkus")
+                .version(HttpVersion.HTTP_1_1)
+                .build();
+
+        final String expectedFormat = "GET /quarkus HTTP/1.1" + System.lineSeparator()
+                + "Cookie: any-cookie" + System.lineSeparator()
+                + "Accept: application/json, text/plain, */*" + System.lineSeparator()
+                + "Connection: keep-alive";
```

Comment:
This may not work in Windows

Reply:
AFAIK, System.lineSeparator() should return the right characters for the current OS. So it would work in Linux and Windows. Am I wrong?

Reply:
Hm, I think you're right. Since this is meant to be used for logging only and not send through the wire that won't be a problem.